blog/content/post/2025-04-14-ram-wipe.md

```diff
@@ -0,0 +1,1278 @@
+---
+title: ram-wipe against RAM attacks
+abstract: 'This post will introduce some of the very popular attacks that target
+electronic devices - the RAM attacks, but the main topic will be the
+verification of ram-wipe software solution protection from the attacks.'
+cover: /covers/ram-wipe.png
+author: daniil.klimuk
+layout: post
+published: true
+date: 2025-04-14
```

Comment:
@DaniilKl let's change the publish date to the actual one

Reply:
```suggestion
date: 2025-05-08
```

Reply:
@DaniilKl and do not forget to change the filename date as well